tests/ExcelProvider.Tests/ExcelProvider.Tests.fs

```diff
@@ -97,41 +98,41 @@ let ``Read Currency as Decimal``() =
 
 [<Test>]
 let ``Empty Text cell should be null``() =
-    let file = DataTypes()
+    let file = DataTypesWithEmptyRow()
```

Comment:
This change was made due to how these tests were structured. Where before, an `Unbounded` `Range` could expect an additional row where we could test for empty values, now with only one row, `Seq.skip 1` will leave us with an empty row, in turn making `Seq.head` throw. 

By altering the file used to the same Excel file, but explicitly grabbing an extra, empty row, tests that started failing after my changes will now pass.